docs/language/overview.md

```diff
@@ -567,7 +567,7 @@ Bar[Int]::c    //Generic type scoped constant
 (Bar + Baz)::c //Conjunction type scoped constant
 ```
 
-In Bosque names are resolved using the lexical scope where they are used starting from the current block, up to arguments, captured variables, type and finally namespace scoping. Shadowing is not permitted on any variables. However, for arguments/locals in a lambda body can be the same as names in the enclosing declaring body (preventing the closure capture [section 5.6 Lambda Constructors](#5.6-Lambda-Constructors)).
+Names in Bosque are resolved using the lexical scope where they are used, starting from the current block, up to arguments, captured variables, type and finally namespace scoping. Shadowing is not permitted on any variables. However, arguments/locals in a lambda body can be the same as names in the enclosing declaring body (preventing the closure capture [section 5.6 Lambda Constructors](#5.6-Lambda-Constructors)).
```

Comment:
The tweaks to the first sentence here may have been frivolous, deleting the stray "for" in the second sentence was the important bit.

Reply:
Definitely better grammar than the original. Thanks.